l10n_it_fatturapa/demo/account_invoice_fatturapa.xml

```diff
@@ -174,7 +174,7 @@
             <field name="company_id" ref="base.main_company"/>
             <field name="type">out_invoice</field>
             <field name="account_id" ref="account.a_pay"/>
-            <field name="date_invoice" eval="time.strftime('%Y')+'-01-07'"/>
+            <field name="date_invoice" eval="'2015-01-07'"/>
```

Comment:
Questa modifica rende coerente i dati demo con i dati dei test, indipendentemente dall'anno in cui il test viene eseguito.
Il problema è che nel 2016 il test fallirà perchè l'anno fiscale 2015 non eisterà e quindi confermando questa fattura verrà generato errore per via del periodo mancante.

Abbiamo quindi 2 strade:
 - modifichiamo dinamicamente il contenuto dell'XML in funzione dell'anno
 - creiamo sempre, se non esiste, l'anno fiscale 2015

Forse la prima sarebbe + corretta ed anche + semplice